docs/girder_config_options.rst

```diff
@@ -192,6 +192,143 @@ By default, item metadata can contain any keys and values.  These can be given b
         # exclusiveMinimum: 0
         # exclusiveMaximum: 10
 
+Image View Presets
+..................
+
+This is used to specify a list of default presets for viewing images in the folder.
+Presets can be customized and saved in the GeoJS Image Viewer.
+To retrieve saved presets, use http://[serverURL]/api/v1/item/[itemID]/internal_metadata/presets.
+You can convert the response to YAML and paste it into the `imagePresets` key in your config file.
+
+Each preset can specify a name, a view mode, an image frame, and style options.
+
+- The name of a preset can be any string which uniquely identifies the preset.
+
+- There are four options for mode:
+    1. Frame control
+        id: 0
+        name: Frame
+    2. Axis control
+        id: 1
+        name: Axis
+    3. Channel Compositing
+        id: 2
+        name: Channel Compositing
+    4. Band Compositing
+        id: 3
+        name: Band Compositing
+
+- The frame of a preset is a 0-based index representing a single frame in a multiframe image.
+  For single-frame images, this value will always be 0.
+  For channel compositing, each channel will have a `framedelta` value which represents distance from this base frame value.
+  The result of channel compositing is multiple frames (calculated via framedelta) composited together.
+
+- The style of a preset is a dictionary with a schema similar to the [style schema for tile retrieval](tilesource_options.rst#style).
+  The value for a preset's style consists of a band definition, where each band may have the following:
+  1. `band`: A 1-based index of a band within the current frame
+  2. `framedelta`: An integer representing distance from the current frame, used for compositing multiple frames together
+  3. `palette`: A hexidecimal string beginning with "#" representing a color to stain this frame
+  4. `min`: The value to map to the first palette value
+  5. `max`: The value to map to the last palette value
+  6. `autoRange`: A shortcut for excluding a percentage from each end of the value distribution in the image. Express as a float.
+
+The YAML below includes some example presets.
+
+::
+
+    ---
+    # If present, each preset in this list will be added to the preset list
+    # of every image in the folder for which the preset is applicable
+    imagePresets:
```

Comment:
I'm thinking the key should be `imageFramePresets` or `framePresets`, and the title of this section of the docs would be `Image Frame Presets` instead of `Image View Presets`.